Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs

```diff
@@ -192,7 +192,7 @@ public class TheGetContentsMethod
             var archive = await github
                 .Repository
                 .Content
-                .GetArchive("alfhenrik", "ScriptCs.OctoKit", ArchiveFormat.Tarball, "dev");
```

Comment:
💩 sorry! I can add it back in, or I can update the test to use the `master` branch instead (as that **should never** be removed 😇